scripts/generate-google3-docs/resource-reference/generated/resource-docs/accesscontextmanager/accesscontextmanagerserviceperimeter.md

```diff
@@ -2227,15 +2227,17 @@ spec:
     egressPolicies:
     - egressFrom:
         identities:
-        - name: serviceperimeterengressdep
+          - serviceAccountRef:
```

Comment:
Not sure where this change comes from?  Is it another PR where we forgot to generate the docs?